apps/web/src/app.tsx

```diff
@@ -189,6 +194,10 @@ function DesktopAppContents() {
               isNavPaneCollapsed: sizes[0] <= 70,
               isListPaneVisible: sizes[1] > 5 // we keep a 5px margin just to be safe
             });
+
+            if (sizes[1] < LIST_PANE_SNAP_SIZE) {
+              navPane.current?.collapse(1, false);
+            }
```

Comment:
When the list pane is closed by dragging, its `collapsed` property is not set to `false`. I have to manually set that here. Still figuring out if this is the right move